uamqp/client.py

```diff
@@ -291,14 +294,19 @@ def mgmt_request(self, message, operation=None, operation_type=None, node='$mana
             if mgmt_link.mgmt_error:
                 raise mgmt_link.mgmt_error
             if mgmt_link.mgmt_link_open_status != ManagementOpenResult.OK:
-                raise AMQPConnectionError("Failed to open mgmt link: {}".format(mgmt_link.mgmt_link_open_status))
+                # TODO: update below with correct status code + info
+                raise AMQPConnectionError(400, "Failed to open mgmt link: {}".format(mgmt_link.mgmt_link_open_status), {})
         operation_type = operation_type or b'empty'
         status, response, description = mgmt_link.execute(
             message,
             operation=operation,
             operation_type=operation_type,
             timeout=timeout
         )
+        status_code_field = kwargs.pop('status_code_field', b'statusCode')
+        description_field = kwargs.pop('description_field', b'statusDescription')
+        response.application_properties[status_code_field] = status
+        response.application_properties[description_field] = description
```

Comment:
we could remove those first..

I think those field should be handled during mgmt link initialization:
https://github.com/Azure/azure-uamqp-python/blob/pyproto/uamqp/management_link.py#L52-L53

let's revisit this in the work for supporting mgmt link